_docs/resources/engaging-helix.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Engaging Helix
+layout: docs
+status: stable
+resource: true
+category: Resources
+last-modified: 2018-07-11
+usage: >
+   The long and short of it is that you should reach out the the Helix team anytime the team might be helpful to you, but it’s not always easy to know what that is. Use the following scenarios as a guide for when you should contact the Helix team.
```

Comment:
I'm not sure what to do with this content, tbh. It's located in an area of the page that most folks probably overlook due to banner blindness. Even so, I would go one of two directions with this content. I would:
1) make it super plain and say something like "Use the following scenarios as a guide..." - short and sweet.
2) expand this text to include more information about Helix and the Helix team. I feel like this page is missing that context. Maybe the 'About' information could be placed further down the page, before the cartoons? There may be new team getting on board, or new hires, that could benefit from that kind of elevator pitch.

Reply:
I agree with Chris' assessment. For the moment, I think we should go with option 1.